lib/testy.js

```diff
@@ -60,7 +60,8 @@ const pending = new PendingMarker(testRunner);
  * @returns {void}
  */
 function test(name, testBody) {
-  testRunner.registerTest(name, testBody, ui.testCallbacks());
+  const registeredTest = testRunner.registerTest(name, testBody, ui.testCallbacks());
+  return registeredTest;
```

Comment:
let's inline this variable because it is used once and update the JSDoc comment (it says it return void, but now the test is returned)